components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -159,18 +160,13 @@ export function CreateWorkspacePage() {
     const [errorIde, setErrorIde] = useState<string | undefined>(undefined);
 
     const existingWorkspaces = useMemo(() => {
-        if (!workspaces.data || !CommitContext.is(workspaceContext.data)) {
+        if (!workspaces.data) {
             return [];
         }
         return workspaces.data.filter(
-            (ws) =>
-                ws.latestInstance?.status?.phase === "running" &&
-                CommitContext.is(ws.workspace.context) &&
-                CommitContext.is(workspaceContext.data) &&
-                ws.workspace.context.repository.cloneUrl === workspaceContext.data.repository.cloneUrl &&
-                ws.workspace.context.revision === workspaceContext.data.revision,
+            (ws) => ws.status?.phase?.name === WorkspacePhase_Phase.RUNNING && ws.contextUrl === contextURL,
```

Comment:
it feels maybe we want to add git info on Workspace? cc @svenefftinge

Reply:
actually here we already have revision from ws.status.gitStatus?.latestCommit and cloneUrl from ws.status.gitStatus?.cloneUrl

why don't we use them instead?